src/gatling/simulations/scenarios/Solicitor_Bails.scala

```diff
@@ -0,0 +1,1383 @@
+        .check(jsonPath("$.case_id").is("${case_Id}"))
+        .check(substring("access_granted")))
+
+      .exec(Common.isAuthenticated)
+
+      .exec(Common.healthcheck("%2Fcases%2Fcase-details%2F${case_Id}%2Ftrigger%2FsubmitApplication%2FsubmitApplicationdeclarationOnSubmit"))
+
+      .exec(getCookieValue(CookieKey("XSRF-TOKEN").withDomain(BaseURL.replace("https://", "")).saveAs("XSRFToken")))
+
+    }
+    .pause(MinThinkTime, MaxThinkTime)
+
+
+    /*======================================================================================
+    * The applicant has confirmed that the facts stated in this application are true.
+    ======================================================================================*/
+
+    .group("XUI_Bails_460_Facts_Are_True") {
+
+      exec(http("XUI_Bails_460_005_Facts_Are_True")
+        .post("/data/case-types/Bail/validate?pageId=submitApplicationdeclarationOnSubmit")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-data-validate.v2+json;charset=UTF-8")
+        .header("x-xsrf-token", "${XSRFToken}")
+        .body(ElFileBody("bodies/bails/BailsFactsAreTrue.json"))
+        .check(substring("isLegalRep")))
+
+      .exec(Common.healthcheck("2Fcases%2Fcase-details%2F${case_Id}%2Ftrigger%2FsubmitApplication%2Fsubmit"))
+
+    }
+    .pause(MinThinkTime, MaxThinkTime)
+
+
+    /*======================================================================================
+    * Submit the application
+    ======================================================================================*/
+
+    .group("XUI_Bails_470_Submit_The_Application") {
+
+      exec(http("XUI_Bails_470_005_Submit_The_Application")
+        .post("/data/cases/${case_Id}/events")
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.create-event.v2+json;charset=UTF-8")
+        .header("x-xsrf-token", "${XSRFToken}")
+        .body(ElFileBody("bodies/bails/BailsSubmitTheApplication.json"))
+        .check(jsonPath("$.callback_response_status").is("CALLBACK_COMPLETED")))
+
+      .exec(Common.healthcheck("%2Fcases%2Fcase-details%2F${case_Id}%2Ftrigger%2FsubmitApplication%2Fconfirm"))
+
+    }
+    .pause(MinThinkTime, MaxThinkTime)
+
+
+  val UploadBailSummary =
+
+    /*======================================================================================
+    * Load case
+    ======================================================================================*/
+
+    group("XUI_Bails_480_Open_Case") {
+
+      exec(Common.healthcheck("%2Fcases%2Fcase-details%2F${case_Id}"))
+
+      .exec(http("XUI_Bails_480_005_Open_Case")
+        .get("/data/internal/cases/${case_Id}")
+        .headers(Headers.navigationHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-case-view.v2+json")
+        .check(jsonPath("$.case_id").is("${case_Id}")))
```

Comment:
maybe check the $.state field if useful

Reply:
Changed it to check the state.name is 'Application submitted'